src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -98,7 +100,16 @@ class ChaptersButton extends TextTrackButton {
       menu.children_.unshift(title);
       Dom.insertElFirst(title, menu.contentEl());
     }
-
+    // We will empty out the children each time because we want a 
+    // fresh new list each time
+    
+    if (menu.contentEl_.childElementCount > 1) {
```

Comment:
these should also be removed from `this.items`, so, probably better to loop through those and dispose each item.